core/tests/gauge-taiko automated tests/voiceover-tests/Dockerfile

```diff
@@ -0,0 +1,54 @@
+# Building the image
```

Comment:
Will this Dockerfile need to be replicated for every test, or is it a general thing that's used for every test?

If the latter, put it in the gauge-taiko folder, not the voiceover-tests subfolder.

Reply:
yes, this can be in the parent folder